redisearch/client_test.go

```diff
@@ -935,6 +938,56 @@ func TestClient_CreateIndex(t *testing.T) {
 	assert.Equal(t, "John", docs[1].Properties["name"])
 }
 
+func TestClient_CreateJsonIndex(t *testing.T) {
+	c := createClient("create-json-index")
+	flush(c)
+	version, err := c.getRediSearchVersion()
+	assert.Nil(t, err)
```

Comment:
The assert is unneccesary given the version check. I'd do version, _ := c.getRediSearchVersion() and remove this line.